### PR TITLE
Adds OracleLinux install

### PR DESCRIPTION
**Pull request by michaelpporter**
Adds support for OracleLinux

### CODE REVIEW

File: tasks/OracleLinux [hunk 1]
```diff
@@ -0,0 +1 @@
+CentOS
```
Comment (jaredledvina) [inline tasks/OracleLinux:1]: Wow, this works?

**Comment by jaredledvina** [comment]
Can we leverage https://hub.docker.com/r/dokken/oraclelinux-7 to get this included with molecule?

File: tasks/OracleLinux (hunk 1 above)
Comment (michaelpporter) [reply tasks/OracleLinux:1]: Yes. We have one host that is OracleLinux we have to install a few things for some of the plugins install: 

    - "gcc-c++"
    - "kernel-devel"
    - "make"

**Comment by michaelpporter** [comment]
> Can we leverage https://hub.docker.com/r/dokken/oraclelinux-7 to get this included with molecule?

I will see what I can do tonight.

**Comment by michaelpporter** [comment]
Hum, there is something different with our host from docker:

Our host
```bash
[root@ip-X-X-X-X ~]# yum install epel-release
Loaded plugins: ulninfo
epel/x86_64/metalink                                                                                                                                                                                                                                                  |  18 kB  00:00:00
epel                                                                                                                                                                                                                                                                  | 4.7 kB  00:00:00
mysql-connectors-community                                                                                                                                                                                                                                            | 2.5 kB  00:00:00
mysql-tools-community                                                                                                                                                                                                                                                 | 2.5 kB  00:00:00
mysql56-community                                                                                                                                                                                                                                                     | 2.5 kB  00:00:00
ol7_UEKR3                                                                                                                                                                                                                                                             | 1.2 kB  00:00:00
ol7_latest                                                                                                                                                                                                                                                            | 1.4 kB  00:00:00
sensu                                                                                                                                                                                                                                                                 | 2.5 kB  00:00:00
(1/3): epel/x86_64/updateinfo                                                                                                                                                                                                                                         | 959 kB  00:00:00
(2/3): sensu/7Server/x86_64/primary_db                                                                                                                                                                                                                                |  87 kB  00:00:00
(3/3): epel/x86_64/primary_db                                                                                                                                                                                                                                         | 6.6 MB  00:00:01
(1/3): ol7_latest/x86_64/group                                                                                                                                                                                                                                        | 810 kB  00:00:00
(2/3): ol7_latest/x86_64/updateinfo                                                                                                                                                                                                                                   | 838 kB  00:00:00
(3/3): ol7_latest/x86_64/primary                                                                                                                                                                                                                                      |  12 MB  00:00:00
ol7_latest                                                                                                                                                                                                                                                                       12252/12252
Resolving Dependencies
--> Running transaction check
---> Package epel-release.noarch 0:7-7 will be updated
---> Package epel-release.noarch 0:7-11 will be an update
--> Finished Dependency Resolution

Dependencies Resolved

=============================================================================================================================================================================================================================================================================================
 Package                                                                    Arch                                                                 Version                                                            Repository                                                          Size
=============================================================================================================================================================================================================================================================================================
Updating:
 epel-release                                                               noarch                                                               7-11                                                               epel                                                                15 k

Transaction Summary
=============================================================================================================================================================================================================================================================================================
Upgrade  1 Package

Total download size: 15 k
Is this ok [y/d/N]: y
Downloading packages:
Delta RPMs disabled because /usr/bin/applydeltarpm not installed.
epel-release-7-11.noarch.rpm                                                                                                                                                                                                                                          |  15 kB  00:00:00
Running transaction check
Running transaction test
Transaction test succeeded
Running transaction
  Updating   : epel-release-7-11.noarch                                                                                                                                                                                                                                                  1/2
  Cleanup    : epel-release-7-7.noarch                                                                                                                                                                                                                                                   2/2
  Verifying  : epel-release-7-11.noarch                                                                                                                                                                                                                                                  1/2
  Verifying  : epel-release-7-7.noarch                                                                                                                                                                                                                                                   2/2

Updated:
  epel-release.noarch 0:7-11

Complete!
```

Docker
```bash
    fatal: [oraclelinux-7]: FAILED! => {"changed": false, "msg": "No package matching 'epel-release' found available, installed or updated", "rc": 126, "results": ["No package matching 'epel-release' found available, installed or updated"]}
```

Looking in to it.

**Comment by michaelpporter** [comment]
@jaredledvina Passes locally, I could not get a OL6 build to work so I think we stick with OL7 the testing.

**Review by jaredledvina — APPROVED** [review]
Looks good! I tried to see if there was something we needed to add to https://github.com/sensu/sensu-ansible/blob/master/meta/main.yml from https://galaxy.ansible.com/api/v1/platforms/?search= but, it looks like EL 6/7 covers Oracle linux per https://github.com/ansible/galaxy/issues/1542

👍

**Pull request merged**